ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -2808,7 +2834,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'P12S Pangenesis Collect',
       type: 'GainsEffect',
       netRegex: { effectId: pangenesisEffectIds },
-      condition: (data) => !data.pangenesisDebuffsCalled && !data.isDoorBoss,
+      condition: (data) => !data.pangenesisDebuffsCalled && data.phase === 'pangenesis',
```

Comment:
I guess I don't understand this change or all of the other ones below.  When do these effects happen in the final boss other than in this phase? Is this a logic change or just cleanup?

Reply:
> When do these effects happen in the final boss other than in this phase? Is this a logic change or just cleanup?

Purely stylistic/cleanup - they don't otherwise happen in the final phase.